lib/seccomp/CSystemCallFilter_Linux.cc

```diff
@@ -35,6 +35,7 @@ const std::uint32_t SECCOMP_DATA_NR_OFFSET = 0x00;
 #define SECCOMP_RET_ERRNO 0x00050000U
 #define SECCOMP_RET_ALLOW 0x7fff0000U
 #define SECCOMP_RET_DATA 0x0000ffffU
+#define SECCOMP_RET_KILL 0x00000000U
```

Comment:
Remove this please

Reply:
I left it in to avoid having to look it up next time we need to debug.  It doesn't really hurt, and can be removed when all the constants are removed once our minimum build machine has the header containing the constants.